tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "out": "docs",
     "excludeExternals": true,
     "name": "Subsocial JS SDK",
-    "theme": "./themes/docs-theme",
+    "customCss": "./themes/main.css",
```

Comment:
Note: With the current typedoc script, all files inside `themes/docs-theme` are not used anymore and instead only extending default theme with `main.css`.

It was probably used as a template for the previous js docs. Maybe we can remove it?
cc @olehmell

Reply:
Yes, lets try, if it works without it so it is cool

Reply:
Okay, this is handled in #114